src/coreComponents/dataRepository/Group.hpp

```diff
@@ -165,6 +165,16 @@ class Group
    */
   string dumpInputOptions() const;
 
+  /**
+   * @brief @return a comma separated string containing all sub groups name.
+   */
+  string dumpSubGroupsNames() const;
+
+  /**
+   * @brief @return a comma separated string containing all wrappers name.
+   */
+  string dumpWrappersNames() const;
```

Comment:
I can let it go like this because the pattern was already in place (`printDataHierarchy`, `dumpInputOptions`) but attaching the display format to the extraction of the data is not what we should be doing.

Reply:
I was thinking the same thing about this. I can stop there with for this PR, but I'm just curious about what you would prefer about where / how we could regroup these 4 methods ? A GroupView class which could reference a Group and expose displaying methods ?